docs/terra/compute.rst

```diff
@@ -30,6 +30,19 @@ VirtualEnvCompute
 
 The :py:class:`terra.compute.virtualenv.Compute` is used to run services locally in a virtualenv. While terra itself runs in a virtualenv, the terra app has its own virtualenv, which typically exists in the container, but exists locally when using this compute. This compute does not use :ref:`settings-path-translation`.
 
+Debugging in the compute
+------------------------
+
+In order to debug in the exact compute environment, two environment were introduced to help:
+
+.. envvar:: TERRA_DEBUG_SERVICE
+
+Since there are many complicated steps that go into starting an environment for a service runner (i.e. starting a docker container with the right mounts and temporary settings file). Setting :envvar:`TERRA_DEBUG_SERVICE` will start the environment and run the command :envvar:`TERRA_DEBUG_SHELL` to allow you to debug in the actual environment with ease. :envvar:`TERRA_DEBUG_SERVICE` must be set to the service runner name or any class in it's class hierarchy (e.g. ``MyServiceRunner_docker``, ``MyServiceRunner``, or to stop on all runners ``object``).
+
+.. envvar:: TERRA_DEBUG_SHELL
+
+The command that is when the service is debugged. Default: ``bash``, but you can set it to anything (e.g. ``bash --rcfile "/test_argument/dir with spaces/my.rc"``).
```

Comment:
```suggestion
The command that is run when the service is debugged. Default: ``bash``, but you can set it to anything (e.g. ``bash --rcfile "/test_argument/dir with spaces/my.rc"``).
```

Reply:
I don't think its necessary to call out "dir with spaces" here

Reply:
Well, it's more likely to happen on windows, and it has to be handled correctly (which `shlex` does, thankfully)